### ISSUE TITLE
Only red Aura LED works--Maybe a bug?

### ISSUE THREAD

**Issue by christianhaitian** [gnuton/asuswrt-merlin.ng#705]
**Router Model Affected**
Models: RT-AX82u

**Firmware Version Affected**
[3004.388.8_2-gnuton1](http://gateway.valeris.lan/Advanced_FirmwareUpgrade_Content.asp)

**Is this bug present in upstream Merlin releases too?**
I don't know

**Describe the bug**
Only the red Aura LED works.  I haven't tested on the stock firmware to see if it's an issue with my specific unit.  I'm checking to see if others have all other LED colors available for Aura is this truly a bug?

**To Reproduce**
Enable Aura LED, set to static and pick a color besides Red.  No other color illuminates on the unit.

**Expected behavior**
All various color combinations should work such as blue.

**Screenshots**
If applicable, add screenshots to help explain your problem or remove this section

**Comment by gnuton** [comment]
Hi thanks for reporting this.
HAve you tried to reset the router after flashing?

**Comment by slythrn** [comment]
I have an AX82U and I don't have an issue at all changing the LED light color.

**Comment by christianhaitian** [comment]
> Hi thanks for reporting this. HAve you tried to reset the router after flashing?

Sorry for the extended time in responding.  I did a couple of factory resets, one via the GUI and another via the hardware reset button and the issue seems to be resolved.  Strange as I did a factory reset initially then configured the router and then noticed this issue but it seems to be good now.  Thanks again for the help and adding support for this router.
